challenges/streams.js

```diff
@@ -0,0 +1,43 @@
+const stream = require("stream");
+
+function setupStreams(dataInputStream, dataOutputStream, callback) {
+  // Write your code here
+  // dataInputStream.pipe(dataOutputStream);
+
+  dataInputStream.on("data", function (chunk) {
+    const jsonData = JSON.parse(chunk.toString());
+    dataOutputStream.write(jsonData);
+    callback(chunk);
+  });
+
+  // dataOutputStream.write(chunk);
+  // callback(chunk)
+  // dataOutputStream.end();
+}
+
+let readable = new stream.Readable();
+let writable = new stream.Writable({
+  objectMode: true,
+  write: (chunk, encoding, callback) => {
+    console.log("chunk: ", chunk);
+    callback(null, true);
+  },
+});
+setupStreams(readable, writable, () => console.log("onEnd"));
+
+readable.push('{ "log": "Request received" }');
+readable.push('{ "lo22g": "Request received" }');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/denpalrius/js-algorithms-and-data-structures/pullRequest?prid=7202625)
```suggestion
readable.push("{ \"lo22g\": \"Request received\" }");
```